src/app/pages/send/send-crypto-asset-form/form/stx/stx-send-form.tsx

```diff
@@ -1,88 +1,47 @@
-import { Outlet, useNavigate } from 'react-router-dom';
-
-import { Box } from '@stacks/ui';
-import { Form, Formik } from 'formik';
-
-import { HIGH_FEE_WARNING_LEARN_MORE_URL_STX } from '@shared/constants';
-import { RouteUrls } from '@shared/route-urls';
-
 import { StxAvatar } from '@app/components/crypto-assets/stacks/components/stx-avatar';
-import { EditNonceButton } from '@app/components/edit-nonce-button';
-import { FeesRow } from '@app/components/fees-row/fees-row';
-import { NonceSetter } from '@app/components/nonce-setter';
-import { HighFeeDrawer } from '@app/features/high-fee-drawer/high-fee-drawer';
 import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
 
 import { AmountField } from '../../components/amount-field';
-import { FormFooter } from '../../components/form-footer';
-import { MemoField } from '../../components/memo-field';
 import { SelectedAssetField } from '../../components/selected-asset-field';
-import { SendCryptoAssetFormLayout } from '../../components/send-crypto-asset-form.layout';
 import { SendFiatValue } from '../../components/send-fiat-value';
 import { SendMaxButton } from '../../components/send-max-button';
-import { StacksRecipientField } from '../../family/stacks/components/stacks-recipient-field';
-import { defaultSendFormFormikProps } from '../../send-form.utils';
+import { StacksCommonSendForm } from '../stacks/stacks-common-send-form';
 import { useStxSendForm } from './use-stx-send-form';
 
 export function StxSendForm() {
-  const navigate = useNavigate();
   const stxMarketData = useCryptoCurrencyMarketData('STX');
 
   const {
     availableStxBalance,
     initialValues,
-    onFormStateChange,
     previewTransaction,
     sendMaxBalance,
-    stxFees,
+    stxFees: fees,
     validationSchema,
   } = useStxSendForm();
 
+  const amountField = (
+    <AmountField
+      balance={availableStxBalance}
+      switchableAmount={<SendFiatValue marketData={stxMarketData} assetSymbol={'STX'} />}
```

Comment:
```suggestion
      switchableAmount={<SendFiatValue marketData={stxMarketData} assetSymbol="STX" />}
```
Would be great if there's a lint rule to enforce string being a plain string

Reply:
hm, couldn't find rule for this